posthog/src/main/java/com/posthog/internal/PostHogRemoteConfig.kt

```diff
@@ -207,14 +217,7 @@ internal class PostHogRemoteConfig(
                             """Feature flags are quota limited, clearing existing flags.
                                     Learn more about billing limits at https://posthog.com/docs/billing/limits-alerts""",
                         )
-                        this.featureFlags = null
```

Comment:
was it intentional that we didnt clear `requestId` here? if so, I should conditionally clear `requestId` or not when calling `clearFlags`

Reply:
@dmarticus question for you since you added this part